go-controller/pkg/ovn/pods.go

```diff
@@ -129,6 +129,13 @@ func (oc *DefaultNetworkController) syncPods(pods []interface{}) error {
 		if err != nil {
 			return fmt.Errorf("could not generate ops to delete stale ports from logical switch %s (%+v)", n.Name, err)
 		}
+		if config.HybridOverlay.Enabled {
+			err = oc.allocateHybridOverlayDRIP(n)
```

Comment:
so we are using the sync pods method to reserve the DRIPs for the nodes in case a node was being retried or something. This guards against the pod handler from adding new pods and potentially taking a reserved address. Would be good to have a comment here explaining that.

Reply:
done